src/compression.jl

```diff
@@ -78,16 +78,6 @@ end
 # Methods
 # -------
 
-function TranscodingStreams.initialize(codec::ZstdCompressor)
-    code = initialize!(codec.cstream, codec.level)
-    if iserror(code)
-        zstderror(codec.cstream, code)
-    end
-    reset!(codec.cstream.ibuffer)
-    reset!(codec.cstream.obuffer)
```

Comment:
Where does this happen now?

Reply:
This happens in `reset!`
https://github.com/JuliaIO/CodecZstd.jl/blob/60079dd3d5d7da7e32cd75f8361c6584fb36fe24/src/libzstd.jl#L73-L74

Which is called in `startproc` https://github.com/JuliaIO/CodecZstd.jl/blob/60079dd3d5d7da7e32cd75f8361c6584fb36fe24/src/compression.jl#L104